Directory.Packages.props

```diff
@@ -3,6 +3,7 @@
     <PackageVersion Include="coverlet.collector" Version="6.0.0" />
     <PackageVersion Include="Fody" Version="6.6.4" />
     <PackageVersion Include="IsExternalInit" Version="1.0.3" />
+    <PackageVersion Include="Microsoft.Bcl.HashCode" Version="1.1.1" />

```

Comment:
Does this work without including the dependency in the NuGet package?

Aka <https://github.com/dotnet/roslyn/blob/main/docs/features/source-generators.cookbook.md#use-functionality-from-nuget-packages>

Reply:
@lsoft, soon. Most likely this week.

Reply:
@GerardSmit, thanks for pointing that out. Will ensure it gets included. BTW, if any of the commits I made after your original PR don't make sense, let me know. We should have this merged soon.

Reply:
Everything else look OK to me; using a ushort instead of a hacky wrapper struct is probably better. 😉